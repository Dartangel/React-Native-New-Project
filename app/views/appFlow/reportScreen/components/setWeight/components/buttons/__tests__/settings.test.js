import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import OwnBtn from '../index';

describe('OwnBtn', () => {
    test('OwnBtn snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<OwnBtn />);
        expect(snapshot).toMatchSnapshot();
    });
});