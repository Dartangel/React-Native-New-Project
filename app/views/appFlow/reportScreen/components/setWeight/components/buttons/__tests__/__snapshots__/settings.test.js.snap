// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OwnBtn OwnBtn snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 0.3,
      "flexDirection": "row",
      "justifyContent": "space-around",
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Text
      style={
        Object {
          "color": "#08DBA5",
        }
      }
    />
  </TouchableOpacity>
</View>
`;
